website/docs/r/vapp_access_control.html.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_access_control"
+sidebar_current: "docs-vcd-resource-vapp-access-control"
+description: |-
+  Provides a vCloud Director Access Control structure for a vApp.
+---
+
+# vcd\_vapp\_access\_control
+
+Provides a vCloud Director Access Control structure for a vApp. This can be used to create, update, and delete access control structures for a vApp.
+
+!> **Warning:** The access control info is tied to a vApp. Thus, there could be only one instance per vApp. Using a different
+definition for the same vApp ID will result in a previous instance to be overwritten.
+
+!> **Note:** access control operations run in tenant context, meaning that, even if the user is a system administrator,
+in every request it uses headers items that define the tenant context as restricted to the organization to which the vApp belongs.
```

Comment:
This is a nice write-up, but after reading it one question rises "why is this important to know?". If we can describe that in a short sentence, would be great.

Also, please change this note to yellow color. No need for it to be red and scary :)
Maybe even the first warning could be yellow?

Reply:
It's useful to know for the ones who need this feature. The others can easily ignore it.